drivers/w1/w1_zephyr_gpio.c

```diff
@@ -289,7 +289,7 @@ static int w1_gpio_init(const struct device *dev)
 
 	if (gpio_is_ready_dt(spec)) {
 		int ret = gpio_pin_configure_dt(spec, GPIO_OUTPUT_INACTIVE | GPIO_OPEN_DRAIN |
-							      GPIO_PULL_UP);
+						      GPIO_INPUT);
```

Comment:
could you please align the GPIO_INPUT to start on the same column as `spec` above.

Reply:
Looking the the latest push, I wonder if it was fine when I wrote the comment. In that case I am sorry.
It's correct to not align it with spec, as it's part of the second argument.